### PR TITLE
[MINOR] Fix some typo

### PR DESCRIPTION
**Pull request by panbingkun**
### What changes were proposed in this pull request?
The pr aims to fix some typo.

### Why are the changes needed?
Fix some typo.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
- Pass GA.
- Manually test.

### Was this patch authored or co-authored using generative AI tooling?
No.

### CODE REVIEW

File: sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/MetadataOperation.java [hunk 1]
```diff
@@ -58,7 +55,7 @@ public void close() throws HiveSQLException {
   }
 
   /**
-   * Convert wildchars and escape sequence from JDBC format to datanucleous/regex
+   * Convert wildcards and escape sequence from JDBC format to datanucleus/regex
```
Comment (HyukjinKwon) [inline sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/MetadataOperation.java:58]: I think this is from Hive. Let's just leave them.

Comment (panbingkun) [reply sql/hive-thriftserver/src/main/java/org/apache/hive/service/cli/operation/MetadataOperation.java:58]: Okay.
I am still checking if there are other similar `typo` and submitting them together.

File: docs/cloud-integration.md [hunk 2]
```diff
@@ -121,12 +121,12 @@ for talking to cloud infrastructures, in which case this module may not be neede
 Spark jobs must authenticate with the object stores to access data within them.
 
 1. When Spark is running in a cloud infrastructure, the credentials are usually automatically set up.
-1. `spark-submit` is able to read the `AWS_ENDPOINT_URL`, `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`
+2. `spark-submit` is able to read the `AWS_ENDPOINT_URL`, `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`
```
Comment (HyukjinKwon) [inline docs/cloud-integration.md:124]: `1. 1. 1.` is fine. we don;t need to manually change the numbering

**Review by HyukjinKwon — APPROVED** [review]
LGTM otherwise

File: docs/cloud-integration.md (hunk 2 above)
Comment (panbingkun) [reply docs/cloud-integration.md:124]: Okay, Let me revert it.

Comment (panbingkun) [reply docs/cloud-integration.md]: Done.

**Comment by HyukjinKwon** [comment]
Merged to master.

**Pull request closed without merging**